zephyr/src/supp_events.h

```diff
@@ -3,6 +3,39 @@
 
 #include <zephyr/net/wifi_mgmt.h>
 
+/* Connectivity Events */
+#define _NET_MGMT_WPA_SUPP_LAYER	NET_MGMT_LAYER_L2
```

Comment:
Lets not try to align the values of the defines at the same level, i.e. keep a single space between the define name and define value. e.g. `#define _NET_MGMT_WPA_SUPP_LAYER NET_MGMT_LAYER_L2`

Reply:
Done.